src/main/java/com/google/jenkins/plugins/storage/AbstractUploadDescriptor.java

```diff
@@ -64,7 +64,7 @@ public UploadModule getModule() {
    * This callback validates the {@code bucketNameWithVars} input field's
    * values.
    */
-  public FormValidation doCheckBucketNameWithVars(
+  public static FormValidation staticDoCheckBucket(
       @QueryParameter final String bucketNameWithVars)
```

Comment:
Consider removing the annotation, since this method is no longer bound to a request.